pygmt/src/text.py

```diff
@@ -0,0 +1,206 @@
+"""
+text - Plot text on a figure figure.
+"""
+import numpy as np
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    build_arg_string,
+    data_kind,
+    dummy_context,
+    fmt_docstring,
+    is_nonstr_iter,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    J="projection",
+    B="frame",
+    C="clearance",
+    D="offset",
+    G="fill",
+    N="no_clip",
+    V="verbose",
+    W="pen",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+    t="transparency",
+)
+@kwargs_to_strings(
+    R="sequence",
+    textfiles="sequence_space",
+    angle="sequence_comma",
+    font="sequence_comma",
+    justify="sequence_comma",
+    p="sequence",
+)
+def text(
```

Comment:
```suggestion
def text_(
```

How about this to fix the pylint `W0621: Redefining name 'text' from outer scope (line 40) (redefined-outer-name)` warning? And then use `from pygmt.src.text import text_ as text` in `pygmt/src/__init__.py`.